### PR TITLE
Fixes #35785 - Add bios fields to reported data

### PR DESCRIPTION
**Pull request by pkoprda**
Adding bios fields to reported data on host index page.
Fields to be added:
 * bios vendor
 * bios release date
 * bios version

### CODE REVIEW

**Review by adamruzicka — APPROVED** [review]
Nothing to add here, ACK

**Comment by adamruzicka** [comment]
Thank you @pkoprda !

File: config/initializers/foreman_register.rb [hunk 1]
```diff
@@ -53,5 +53,11 @@
     add_pagelet :hosts_table_column_content, key: :disks_total, callback: ->(host) { humanize_bytes(host.reported_data&.disks_total) }, class: common_td_class
     add_pagelet :hosts_table_column_header, key: :kernel_version, label: _('Kernel version'), width: '12%', class: common_th_class
     add_pagelet :hosts_table_column_content, key: :kernel_version, callback: ->(host) { host.reported_data&.kernel_version }, class: common_td_class
+    add_pagelet :hosts_table_column_header, key: :bios_vendor, label: _('Bios vendor'), width: '8%', class: common_th_class
+    add_pagelet :hosts_table_column_content, key: :bios_vendor, callback: ->(host) { host.reported_data&.bios_vendor }, class: common_td_class
+    add_pagelet :hosts_table_column_header, key: :bios_release_date, label: _('Bios release date'), width: '10%', class: common_th_class
+    add_pagelet :hosts_table_column_content, key: :bios_release_date, callback: ->(host) { host.reported_data&.bios_release_date }, class: common_td_class
+    add_pagelet :hosts_table_column_header, key: :bios_version, label: _('Bios version'), width: '12%', class: common_th_class
```
Comment (ekohl) [inline config/initializers/foreman_register.rb:60]: Late to the party, but it's BIOS and not Bios: https://github.com/theforeman/foreman/pull/9537

**Review by ekohl — COMMENTED** [review]
I considered this for a cherry pick, but it introduces new strings which puts additional burden on translators. We don't really have a good process for introducing new strings into stable releases so I'm a bit on the fence and leaning to postponing to a future release.

While reviewing I noticed another issue in the strings.

**Pull request merged**